ui-tests/src/test/java/org/openmrs/reference/page/PatientCaptureVitalsPage.java

```diff
@@ -21,6 +21,7 @@
     private static final By CONFIRM_BUTTON = By.xpath("//ul[@id='formBreadcrumb']/li[2]/span");
     private static final By CONFIRM_BUTTON_2 = By.id("coreapps-vitals-confirm");
     private static final By SAVE_BUTTON = By.xpath("//button[@type='submit']");
+    private static final By SAVE_CHANGES_BUTTON = By.xpath("//input[@value='Save Changes']");
```

Comment:
What happens when we switch locale to something other than English? Will the save button text still be `Save Changes`?

Reply:
It will definitely change!, Let me fix this immediately. Thanks @dkayiwa  for pointing this out.

Reply:
Hello @dkayiwa please spare some time to review my changes.